collector/nginx_plus.go

```diff
@@ -42,10 +48,24 @@ func NewNginxPlusCollector(nginxClient *plusclient.NginxClient, namespace string
 			"received":      newServerZoneMetric(namespace, "received", "Bytes received from clients", nil),
 			"sent":          newServerZoneMetric(namespace, "sent", "Bytes sent to clients", nil),
 		},
+		streamServerZoneMetrics: map[string]*prometheus.Desc{
+			"processing":     newStreamServerZoneMetric(namespace, "processing", "Client connections that are currently being processed", nil),
+			"connections":    newStreamServerZoneMetric(namespace, "connections", "Total connections", nil),
+			"sessions_2xx":   newStreamServerZoneMetric(namespace, "sessions", "Total sessions completed", prometheus.Labels{"code": "2xx"}),
+			"sessions_4xx":   newStreamServerZoneMetric(namespace, "sessions", "Total sessions completed", prometheus.Labels{"code": "4xx"}),
+			"sessions_5xx":   newStreamServerZoneMetric(namespace, "sessions", "Total sessions completed", prometheus.Labels{"code": "5xx"}),
+			"sessions_total": newStreamServerZoneMetric(namespace, "sessions", "Total sessions completed", prometheus.Labels{"code": "total"}),
```

Comment:
not required. the total number can be obtained in Prometheus by aggregating 2xx,4xx,5xx
Also,  see responses metrics for http zones